### PR TITLE
Update Redshift to 1.12

### PR DESCRIPTION
**Pull request by Mno-hime**

### CODE REVIEW

File: components/x11/redshift/Makefile [hunk 1]
```diff
@@ -37,15 +41,15 @@ CONFIGURE_ENV	+= XGETTEXT=/usr/gnu/bin/xgettext
 CONFIGURE_ENV	+= MSGFMT=/usr/gnu/bin/msgfmt
 CONFIGURE_ENV	+= PYTHON=$(PYTHON)
 
-CONFIGURE_OPTIONS	+= am_cv_python_pythondir="$(PYTHON_VENDOR_PACKAGES)"
+CONFIGURE_OPTIONS	+= am_cv_python_pythondir="$(PYTHON_VENDOR_PACKAGES.32)"
```
Comment (alarcher) [inline components/x11/redshift/Makefile:45]: Any reason why 32-bit Python is used?

Comment (Mno-hime) [reply components/x11/redshift/Makefile:45]: It looks odd, but with `64/` in the path `gmake REQUIRED_PACKAGES` fails with:
```
/usr/bin/pkgdepend resolve  -m /export/home/newman/ws/oi-userland/components/x11/redshift/build/manifest-i386-redshift.depend
/export/home/newman/ws/oi-userland/components/x11/redshift/build/manifest-i386-redshift.depend has unresolved dependency '
    depend type=require fmri=__TBD \
        pkg.debug.depend.file=redshift_gtk/__init__.py \
        pkg.debug.depend.reason=usr/bin/redshift-gtk \
        pkg.debug.depend.type=python \
        pkg.debug.depend.path=export/home/newman/.local/lib/python3.4/site-packages \
        pkg.debug.depend.path=usr/bin \
        pkg.debug.depend.path=usr/lib/python3.4 \
        pkg.debug.depend.path=usr/lib/python3.4/lib-dynload \
        pkg.debug.depend.path=usr/lib/python3.4/plat-sunos5 \
        pkg.debug.depend.path=usr/lib/python3.4/site-packages \
        pkg.debug.depend.path=usr/lib/python3.4/vendor-packages \
        pkg.debug.depend.path=usr/lib/python34.zip'.
/export/home/newman/ws/oi-userland/components/x11/redshift/build/manifest-i386-redshift.depend has unresolved dependency '
    depend type=require fmri=__TBD \
        pkg.debug.depend.reason=usr/bin/redshift-gtk \
        pkg.debug.depend.type=python \
        pkg.debug.depend.file=64/statusicon.abi3.so \
        pkg.debug.depend.file=64/statusicon.cpython-34m.so \
        pkg.debug.depend.file=64/statusicon.so \
        pkg.debug.depend.file=statusicon.abi3.so \
        pkg.debug.depend.file=statusicon.cpython-34m.so \
        pkg.debug.depend.file=statusicon.py \
        pkg.debug.depend.file=statusicon.pyc \
        pkg.debug.depend.file=statusicon.pyo \
        pkg.debug.depend.file=statusicon.so \
        pkg.debug.depend.file=statusicon/__init__.py \
        pkg.debug.depend.path=export/home/newman/.local/lib/python3.4/site-packages/redshift_gtk \
        pkg.debug.depend.path=usr/bin/redshift_gtk \
        pkg.debug.depend.path=usr/lib/python3.4/lib-dynload/redshift_gtk \
        pkg.debug.depend.path=usr/lib/python3.4/plat-sunos5/redshift_gtk \
        pkg.debug.depend.path=usr/lib/python3.4/redshift_gtk \
        pkg.debug.depend.path=usr/lib/python3.4/site-packages/redshift_gtk \
        pkg.debug.depend.path=usr/lib/python3.4/vendor-packages/redshift_gtk \
        pkg.debug.depend.path=usr/lib/python34.zip/redshift_gtk'.
```
There's comment by @pyhalov https://github.com/OpenIndiana/oi-userland/blob/1ed47a2077bfb7e671c15a56afc3d9451fba23d7/components/library/speech-dispatcher/Makefile#L50 basically saying "this is OK, if we don't ship 64-bit binary modules there".

My understanding is that `$(PYTHON_VENDOR_PACKAGES.32)` is for 32-bit modules and noarch modules, 64-bit only for binary modules.

In any case the component works.

Comment (alarcher) [reply components/x11/redshift/Makefile:45]: Just confirm with @pyhalov then merge it then.

Comment (pyhalov) [reply components/x11/redshift/Makefile:45]: that's fine. python3.4 and 2.7 will use py/pyc modules from /usr/lib/pythonX.Y/vendor-packages. /usr/lib/pythonX.Y/vendor-packages/64 is just for binary modules.

Comment (Mno-hime) [reply components/x11/redshift/Makefile:45]: Cool, though can't merge on my own.

**Pull request merged**